wsf_scraping/spiders/gov_spider.py

```diff
@@ -84,18 +78,14 @@ def save_pdf(self, response):
                 return
 
         # Download PDF file to /tmp
-        filename = urlparse(response.url).path.split('/')[-1]
-        if filename:
-            with open('/tmp/' + filename, 'wb') as f:
-                f.write(response.body)
-            # Populate a WHOArticle Item
-            gov_article = GovArticle({
-                    'title': response.meta.get('title', ''),
-                    'uri': response.request.url,
-                    'pdf': filename,
-                    'sections': {},
-                    'keywords': {}
-                }
-            )
+        filename = self._save_file(response.url, response.body)
+        gov_article = GovArticle({
+                'title': response.meta.get('title', ''),
+                'uri': response.request.url,
+                'pdf': filename,
+                'sections': {},
```

Comment:
would it be possible in the future to add default values for optional fields?

Reply:
I'll create an issue for it 👍